src/components/ExitGateCard/_exitGateCard.scss

```diff
@@ -0,0 +1,96 @@
+@use '#scss/allPartials' as *;
+
+.#{$px}-exit-gate-card {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  padding: 0 $spacing-md;
+
+  @media (max-width: $breakpoint-md) {
+    flex-direction: column;
+
+    &__desktop_image {
+      overflow: visible;
```

Comment:
do we need some kind of `max-width` on the image?  I couldn't tell from the Figma